src/containerapp/azext_containerapp/tests/latest/test_containerapp_preview_scenario.py

```diff
@@ -57,38 +44,50 @@ def test_containerapp_preview_environment_type(self, resource_group):
                                  f' --configuration-settings "appsNamespace=appplat-ns"'
                                  f' --configuration-settings "clusterName={connected_cluster_name}"'
                                  f' --configuration-settings "envoy.annotations.service.beta.kubernetes.io/azure-load-balancer-resource-group={resource_group}"').get_output_in_json()
-            custom_location_name = "my-custom-location"
-            custom_location_id = self.cmd(f'az customlocation create -g {resource_group} -n {custom_location_name} -l {TEST_LOCATION} --host-resource-id {connected_cluster_id} --namespace appplat-ns -c {extension["id"]}').get_output_in_json()['id']
-        except Exception as e:
+            self.cmd(f'az customlocation create -g {resource_group} -n {custom_location_name} -l {TEST_LOCATION} --host-resource-id {connected_cluster_id} --namespace appplat-ns -c {extension["id"]}')
+        except:
             pass
 
         # create connected environment with client or create a command for connected?
         sub_id = self.cmd('az account show').get_output_in_json()['id']
-
+        static_ip = '1.1.1.1'
         connected_env_name = 'my-connected-env'
-        connected_env_resource_id = f"/subscriptions/{sub_id}/resourceGroups/{resource_group}/providers/Microsoft.App/connectedEnvironments/{connected_env_name}"
-        file = f"{resource_group}.json"
-        env_payload = '{{ "location": "{location}", "extendedLocation": {{ "name": "{custom_location_id}", "type": "CustomLocation" }}, "Properties": {{}}}}' \
-            .format(location=TEST_LOCATION, custom_location_id=custom_location_id)
-        write_test_file(file, env_payload)
-        self.cmd(f'az rest --method put --uri "{connected_env_resource_id}?api-version=2022-06-01-preview" --body "@{file}"')
-        containerapp_env = self.cmd(f'az rest --method get --uri "{connected_env_resource_id}?api-version=2022-06-01-preview"').get_output_in_json()
-        while containerapp_env["properties"]["provisioningState"].lower() != "succeeded":
-            time.sleep(5)
-            containerapp_env = self.cmd(
-                f'az rest --method get --uri "{connected_env_resource_id}?api-version=2022-06-01-preview"').get_output_in_json()
+        custom_location_id = f"/subscriptions/{sub_id}/resourceGroups/{resource_group}/providers/Microsoft.ExtendedLocation/customLocations/{custom_location_name}"
+        self.cmd(f'containerapp connected-env create -g {resource_group} --name {connected_env_name} --custom-location {custom_location_name} --static-ip {static_ip} -d "InstrumentationKey=TestInstrumentationKey;IngestionEndpoint=https://ingestion.com/;LiveEndpoint=https://abc.com/" -l {TEST_LOCATION}', checks=[
```

Comment:
seperate to connected_env_scenario_test

Reply:
I will seperate to connected_env_scenario_test in next PR.